index.js

```diff
@@ -121,6 +125,26 @@ function doTransform(k) {
       );
     } else if (ts.isImportClause(n) && n.isTypeOnly) {
       return ts.createImportClause(n.name, n.namedBindings);
+    } else if (ts.isTypeReferenceNode(n) && n.typeName.escapedText === "Omit") {
+      const symbol = checker.getSymbolAtLocation(n.typeName);
+
+      if (
+        symbol !== undefined &&
+        symbol.declarations.length > 0 &&
```

Comment:
(and this could be just `symbol.declarations.length &&`, but the current code is fine too)

Reply:
I changed both expressions.